python/setup_project/storage_map_widget.py

```diff
@@ -453,15 +467,83 @@ def _create_new_storage(self):
             # will switch away from "create" back to "choose" or a storage.
             self.guess_storage()
 
-    def _on_path_changed(self, path, edit_lookup):
+    def _on_path_changed(self, path, platform):
         """
-        Keep track of any path edits as they happen. Kepe the user informed if
+        Keep track of any path edits as they happen. Keep the user informed if
         there are any concerns about the entered text.
         """
 
-        # store the edited path in the appropriate path lookup
+        # does the path only contain slashes?
+        only_slashes = path.replace("/", "") == "" or \
```

Comment:
what about mixed slashes? Maybe change to `only_slashes = path.replace("/", "").path.replace("\\", "") == ""`

Reply:
good call. done.